### PR TITLE
App Icon and Splash Screen 

### LINKED ISSUE #12: Create a Splash Screen for the app
Splash screen improves the UX and is much needed in the modern era apps. 

-- Would like to work on this.

### PR DESCRIPTION
**Pull request by Sarthak2601**
Screenshot_2019-12-05-00-23-34-898_com example anmol courtcounter
Fixes #12 and #14 

*Splash Screen added. 
*Handled splash through Manifest file. 
*App icon set to the app logo in drawables. 

*Removed changes in .idea files in second commit.

### CODE REVIEW

**Review by chanmol1999 — CHANGES_REQUESTED** [review]
Remove changes in .idea files and add a gif of the splash screen.

**Comment by Sarthak2601** [comment]
Removed changes in second commit. 
Added gif in the original PR.

File: Scorecounter/.idea/gradle.xml [hunk 1]
```diff
@@ -13,7 +12,7 @@
           </set>
         </option>
         <option name="resolveModulePerSourceSet" value="false" />
-       
+        <option name="testRunner" value="PLATFORM" />
```
Comment (chanmol1999) [inline Scorecounter/.idea/gradle.xml]: Can you remove this line.

**Review by chanmol1999 — CHANGES_REQUESTED** [review]

**Comment by Sarthak2601** [comment]
Changes done.

**Comment by chanmol1999** [comment]
Resolve the merge conflicts.

**Comment by Sarthak2601** [comment]
@chanmol1999 I don't know why it is conflicting, the updated Gradle.xml file is exactly same as the original one. Please check.

**Comment by chanmol1999** [comment]
> @chanmol1999 I don't know why it is conflicting, the updated Gradle.xml file is exactly same as the original one. Please check.

Why don't you remove that file from your commit.
Just delete that file and push the changes.

**Comment by Sarthak2601** [comment]
@chanmol1999 Done.

**Review by chanmol1999 — APPROVED** [review]

**Pull request merged**